prime-dotnet-webapi/Services/VerifiableCredentialService.cs

```diff
@@ -72,6 +72,8 @@ public class VerifiableCredentialService : BaseService, IVerifiableCredentialSer
         public async Task<JObject> CreateConnectionAsync(Enrollee enrollee)
         {
             var alias = enrollee.Id.ToString();
+            var ISSUER_DID = await _verifiableCredentialClient.GetIssuerDidAsync();
```

Comment:
Should probably be issuerDid since it isn't a constant when you're pulling it from their API, and similarly for schemaId